From 5c507d2993ffea964f7c2b84d77056fad43fc577 Mon Sep 17 00:00:00 2001
From: Hristo Venev <hristo@venev.name>
Date: Wed, 19 Aug 2020 17:05:53 +0300
Subject: [PATCH 290/634] staging: media: rpivid: Select MEDIA_CONTROLLER and
 MEDIA_CONTROLLER_REQUEST_API

MEDIA_CONTROLLER_REQUEST_API is a hidden option. If rpivid depends on it,
the user would need to first enable another driver that selects
MEDIA_CONTROLLER_REQUEST_API, and only then rpivid would become available.

By selecting it instead of depending on it, it becomes possible to enable
rpivid without having to enable other potentially unnecessary drivers.

Signed-off-by: Hristo Venev <hristo@venev.name>
---
 drivers/staging/media/rpivid/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/media/rpivid/Kconfig b/drivers/staging/media/rpivid/Kconfig
index 58b470c80db1..304c3edf0e71 100644
--- a/drivers/staging/media/rpivid/Kconfig
+++ b/drivers/staging/media/rpivid/Kconfig
@@ -3,9 +3,9 @@
 config VIDEO_RPIVID
 	tristate "Rpi H265 driver"
 	depends on VIDEO_DEV && VIDEO_V4L2
-	depends on MEDIA_CONTROLLER
 	depends on OF
-	depends on MEDIA_CONTROLLER_REQUEST_API
+	select MEDIA_CONTROLLER
+	select MEDIA_CONTROLLER_REQUEST_API
 	select VIDEOBUF2_DMA_CONTIG
 	select V4L2_MEM2MEM_DEV
 	help
-- 
2.33.1

